package.json

```diff
@@ -149,8 +149,8 @@
     "puppeteer": "^1.20.0",
     "react": "^16.2.0",
     "react-dom": "^16.2.0",
-    "react-router": "^4.2.0",
-    "react-router-dom": "^4.2.2",
+    "react-router": "^5.2.0",
```

Comment:
path array is supported from version 5, so updated to latest to test. We dont use any breaking changes so should be good here.